cli/types/cypress-npm-api.d.ts

```diff
@@ -8,10 +8,22 @@
 
 declare namespace CypressCommandLine {
   type HookName = 'before' | 'beforeEach' | 'afterEach' | 'after'
+
+  interface CodeFrame {
+    line?: number
+    column?: number
+    originalFile: string
+    relativeFile: string
+    absoluteFile: string
+    frame: string
+    language: string
+  }
+
   interface TestError {
     name: string
     message: string
     stack: string
+    codeFrame?: CodeFrame
```

Comment:
so should this be in the changelog?

Reply:
since it affects public types

Reply:
Yeah, I guess the type changes weren't necessary after all, so no need to add anything to the changelog.